Content.Client/Administration/UI/AdminMenuWindow.xaml.cs

```diff
@@ -23,6 +25,15 @@ public AdminMenuWindow()
             MasterTabContainer.SetTabTitle(5, Loc.GetString("admin-menu-panic-bunker-tab"));
             MasterTabContainer.SetTabTitle(6, Loc.GetString("admin-menu-players-tab"));
             MasterTabContainer.SetTabTitle(7, Loc.GetString("admin-menu-objects-tab"));
+            MasterTabContainer.OnTabChanged += TabChanged;
+        }
+
+        private void TabChanged(int tabIndex)
+        {
+            if (tabIndex == 7)
```

Comment:
Can you make this a constant or an enum and use it in the SetTabTitle so that if it gets changed in the future, it's easier to keep them synced?